trappy/ftrace.py

```diff
@@ -465,6 +472,33 @@ def __parse_trace_file(self, trace_file):
             raise ValueError('Failed to parse ftrace file {}:\n{}'.format(
                 trace_file, str(e)))
 
+    def __getattr__(self, attr):
```

Comment:
IIRC this is something that's not made explicit in the commit message - this raises an exception when trying to get a dataframe for an event by specifying its `unique_word` instead of its `name`. #276 instead raised an exception at parse time, which is different, unless I missed something?

Reply:
I may need to update the commit message, but the behavior is largely the same. #276 did not raise exception at parse time, it was only when an attribute was accessed on the thing retrieved on the trace. Now, it fails straight away when you try to get "the thing":
```
# Used to work
trace.my_unique_word
# Used to raise
trace.my_unique_word.data_frame

# Now raises
trace.my_unique_word
trace.my_unique_word.data_frame
```

Reply:
Oh ok, I missed the `set_attr` at the end of `__add_warnings` in #276. LGTM then.